fmfexporter/adapters/polarion/polarion_test_case.py

```diff
@@ -110,6 +111,8 @@ def from_fmf_testcase(fmf_testcase: FMFTestCase):
         # Tags
         tc.tags = fmf_testcase.tags
 
+        # Verifies
+        tc.verifies = fmf_testcase.verifies
```

Comment:
@michalxo right now if you look above at line 57, you'll see that verifies is currently assigned as concatenation of two dicts: "fmf_testcase.requirements + fmf_testcase.defects".

Later this will become a "linked-work-items" element into the XML to Polarion.

Do you think it does not work?

If we need a separate element in the FMF test case like "verifies", then we must update the FMF schema first.

Reply:
I haven't found defects/requirements. Let me try it first.

Reply:
```2019-10-29 08:12:09,928 ERROR ImportTestcaseThread_124 - Configured workflow condition 'LinkedWorkItem' is not met because there is no linked Work Item of type 'Requirement' with link role of 'verifies'. 
Workflow transition cannot occur until this condition is satisfied.
2019-10-29 08:12:09,928 ERROR ImportTestcaseThread_124 - Unable to set work item status to 'approved', currently 'proposed'.
2019-10-29 08:12:09,929 ERROR ImportTestcaseThread_124 - Rolling back test case import due to earlier errors.```

```
requirements:
          - jira: 'ENTMQBR-0000'
            polarion: 'AMQ-84'
            customer-scenario: True
```
either it does not work, or I am doing something wrong @fgiorgetti